website/docs/cloud-docs/api-docs/workspaces.mdx

```diff
@@ -2893,6 +2893,183 @@ No response body.
 
 Status code `204`.
 
+## Show data retention policy
+
+<EnterpriseAlert>
+This endpoint is exclusive to Terraform Enterprise, and not available in Terraform Cloud.
+</EnterpriseAlert>
+
+`GET /workspaces/:workspace_id/relationships/data-retention-policy`
+
+| Parameter            | Description                                                         |
+| ---------------------| --------------------------------------------------------------------|
+| `:workspace_id`      | The name of the workspace to show the data retention policy for. Obtain this from the [workspace settings](/terraform/cloud-docs/workspaces/settings) or the [Show Workspace](#show-workspace) endpoint. |
+
+
+This endpoint shows the data retention policy set explicitly on the workspace.
+When no data retention policy is set on the workspace, the organization data retention policy will be used by default. Read more about [workspace data retention policies](/terraform/cloud-docs/workspaces/settings/deletion#data-retention-policies).
+
+For more information on the types of data retention policies, refer to [Data Retention Policy Types](/terraform/cloud-docs/api-docs/data-retention-policies#types-of-data-retention-policies)
+
+| Status  | Response                                        | Reason                                                        |
+| ------- | ----------------------------------------------- | ------------------------------------------------------------- |
+| [200][] | [JSON API document][] (`type: "data-retention-policy-delete-olders" or "data-retention-policy-dont-deletes"`) | The request was successful. [Types of Data Retention Policies](/terraform/cloud-docs/api-docs/data-retention-policies#types-of-data-retention-policies). |
+| [404][] | [JSON API error object][]                       | Workspace not found, data retention policy does not exist, or user unauthorized to perform action |
```

Comment:
Non-blocking Nitpick (same as above!): Should we add a period here for consistency?:
```suggestion
| [404][] | [JSON API error object][]                       | Workspace not found, data retention policy does not exist, or user unauthorized to perform action. |
```